indico/modules/search/internal.py

```diff
@@ -17,11 +17,11 @@
 
 
 class InternalSearch(IndicoSearchProvider):
-    def search(self, query, user=None, page=None, object_types=(), **params):
+    def search(self, query, user=None, page=None, object_types=(), allow_admin=False, **params):
```

Comment:
I'd make `allow_admin` keyword-only (also in the base class and other subclasses)